Products/CMFPlone/PloneTool.py

```diff
@@ -34,7 +34,7 @@
                         ManagePortal, ManageUsers, ModifyPortalContent, View
 from Products.CMFCore.interfaces import IDublinCore, IMutableDublinCore
 from Products.CMFCore.WorkflowCore import WorkflowException
-from Products.CMFDefault.DublinCore import DefaultDublinCoreImpl
+from .DublinCore import DefaultDublinCoreImpl
```

Comment:
Minor thing: plone.api convention says: do not use relative imports:
http://ploneapi.readthedocs.org/en/latest/contribute/conventions.html#about-imports